<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modifier un champ JSON</string>
  </property>
  <widget class="QPushButton" name="pushButtonModify">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>370</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Modifier</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>370</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushbuttonAddJsonField">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>10</y>
     <width>28</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonRemoveJsonField">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>28</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelJsonField">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Champ JSON</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelField_1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>150</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Champ_1</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelField_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>150</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Champ_2</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelField_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>150</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Champ_3</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelField_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>150</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Champ_4</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelField_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>150</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Champ_5</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelField_6">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>150</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Champ_6</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelField_7">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>150</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Champ_7</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelField_8">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>150</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Champ_8</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelField_9">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>280</y>
     <width>150</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Champ_9</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelField_10">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>310</y>
     <width>150</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Champ_10</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>250</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>250</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>250</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_6">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>250</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_7">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>220</y>
     <width>250</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_8">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>250</y>
     <width>250</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_10">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>310</y>
     <width>250</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>250</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_1">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>250</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_9">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>280</y>
     <width>250</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
